### ISSUE TITLE
Bcc and CC

### ISSUE THREAD

**Issue by MuhammadMuneeb** [Garethp/php-ews#105]
Whenever I add toBccRecipient or toCcRecipient, I get this error:

> EmailAddress or ItemId must be included in the request.

How is this fixeable?

**Comment by Garethp** [comment]
Can you post the code you're using to add CC and BCC

**Comment by MuhammadMuneeb** [comment]
This is how I am doing it, I am assuming I m missing out something:

```
        $mail = MailAPI::withUsernameAndPassword(Constants::SERVER_URL, Constants::USERNAME, Constants::PASSWORD);
        $message = new MessageType();
        $message->setSubject('Review for Instructions');
        $message->setToRecipients($gl_email);
        $message->setCcRecipients([$email_manager, $email_head]);
        $message->setBody($comment);
        $mail->sendMail($message);

```

**Comment by MuhammadMuneeb** [comment]
Any update on this?

**Comment by Garethp** [comment]
Hi, sorry about that. Try updating to the newly released `0.9.5`, it should solve your issue

**Comment by MuhammadMuneeb** [comment]
Thanks, that worked
